### DISCUSSION TITLE
how to create bulk extension

### DISCUSSION THREAD

**Discussion by home4lab** [mikopbx/Core#388] [Q&A] [answered]
is possible to create bulk of extension instead of manually add one by one in web gui ?
![image](https://user-images.githubusercontent.com/87222176/184057432-b3e8257a-ad78-44ac-b6a1-5d8aeba49c88.png)

**Comment by boffart** [comment, marked as answer]
Use **BackupModule** 
https://wiki-mikopbx-ru.translate.goog/module-backup?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=ru&_x_tr_pto=wapp#zagruzka_dannyx_iz_csv
